src/game/lara/lara_misc.c

```diff
@@ -579,7 +579,7 @@ bool Lara_LandedBad(ITEM_INFO *item, COLL_INFO *coll)
             / (DAMAGE_LENGTH * DAMAGE_LENGTH);
     }
 
-    if (item->hit_points < 0) {
+    if (item->hit_points <= 0) {
```

Comment:
Let's simplify it further like this: `return item->hit_points <= 0`.
Additionally please adjust the commit message to conform with our guidelines, in this case I suggest:
`lara: fix dying anim when landing on low health`

Other than that, IDK, I think this is a fun bug and I'd leave it as-is (0.1% chance to see a secret sequence vs just a regular crunch) but that's just me.

Reply:
Here's how it was done in [TR3](https://github.com/Trxyebeep/tomb3/blob/f06647a8d474cd6c8e3ffb4440d5cfd933816f3f/tomb3/game/lara.cpp#L241).

I'd probably follow that which matches rr's suggestion and classify this is an OG bug. It's harmless though so it's up to rr.